docs/design/fabric-v2+/fpc-registration.puml

```diff
@@ -32,197 +35,203 @@ note over ERCC1, ERCC2 #FFaa00
+  end note
 
-end
+group Org-Enclave binding/certification
+  note right ECC1
+    Here the Org takes "ownership" of this enclave by generating a certificate.
+    In Post-MVP, this certificate is used by an FPC endorsement plugin,
+    to replace the normal peer endorsement signature with the enclave signature.
+  end note
 
-group Org-Enclave binding/certification (to be implemented)
-  Admin -> E_Manager ++: certify
-  E_Manager -> ECC1 ++: getCSR
   ECC1 -> Enclave1 ++: getCSR
   Enclave1 -> Enclave1 : csr <- generate_csr(Private_Keys.Enclave_SK)
   return csr /' ECC1 -> Enclave1 '/
-  return csr /' E_Manager -> ECC1 '/
-  E_Manager -> E_Manager : generate peer certificate for enclave (certified by Org_1)
-  E_Manager -> E_Manager : store Certificate
-  return
-  note over Admin, E_Manager #FFaa00
-    -Org_1 takes "ownership" of this enclave,
-    -the certificate should be made available to the chaincode endorsement plugin,
-     so to include it in the responses (for eventual validation) 
-  end note
+  ECC1 -> ECC1 : generate peer certificate for enclave (certified by the Org)
+  ECC1 -> ECC1 : store Certificate
+    note right ECC1
+      FPC_stub stores the binding certificate with the help of
+      the external builder on the peer local storage but also
+      returns it to the Peer_CLI.
+    end note
 end group
+return Credentials, Certificate
 
-group enclave registration
-  activate Admin
-  Admin -> E_Manager ++: register
-  
-  group invoke ERCC register
-  E_Manager -> ERCC1 ++: register Credentials, Certificate
+end
 
-  note over ERCC1 #FFaa00
-    The evidence check ensures that the evidence is signed by IAS
-    and covers the attested data.
-  end note
-  ERCC1 -> ERCC1 : check Credentials.Evidence
-  ERCC1 -> ERCC1 : check Credentials.Attested_Data.CC_Parameters (Channel_Id, MSPID of sender, TLCC_MREnclave)
-  ERCC1 -> ERCC1 : check Certificate
+  loop for all/"enough" endorsers
+    Peer_CLI -> ERCC1 ++: registerEnclave (Credentials, Certificate)
+    note right ERCC1
+        Note that ERCC aborts if a single check failed.
+    end note
+
+    group enclave attestation - evidence validation
+      ERCC1 -> ERCC1 : check Credentials.Evidence
+      note right ERCC1
+        The evidence check ensures that the evidence is correct (e.g. in
+        case of EPID-based attestation the evidence must be signed by IAS)
+        and covers the attested data.
+
+        Note that the IAS verification key is hardcoded in ERCC. This
+        verification key does not change frequently. In the case the verification
+        key changes, the ERCC must be updated and upgraded through the normal
+        chaincode lifecycle upgrade operation.
+      end note
+      ERCC1 -> ERCC1 : check Credentials.Attested_Data match CC_Parameters known to ERCC
+      note right ERCC1
+        In particular Channel_Id, MSPID of sender, TLCC_MRENCLAVE is verified
+      end note
+   end group
+
+  ERCC1 -> ERCC1 : check Org-Enclave binding certificate
```

Comment:
This is also a post-MVP item, so might have to be correspondingly marked? (But see also above)